### ISSUE TITLE
Dokku unable to find and open file plugin.toml

### ISSUE THREAD

**Issue by mikart143** [dokku/dokku#3764]
### Steps to Reproduce

1. Fresh install of Dokku on CentOS 7
2. Add ssh-key and global domain
3.Push project to newly created app on dokku

#### Actual Results
During building of app error occure: remote: 2019/10/30 22:31:39 open /var/lib/dokku/plugins/available/build-env/plugin.toml: no such file or directory   
 Job is finishing with success but i am not able to see my website :(
#### Expected Results
Working Docker image with my website
### `dokku report APP_NAME` output

[https://pastebin.com/UxGRsh1y](https://pastebin.com/UxGRsh1y)

### Additional information

- Container Inspect Output (if applicable) via `dokku ps:inspect APP_NAME` --- [https://pastebin.com/0af0ja8S](https://pastebin.com/0af0ja8S)

- `cat /home/dokku/<app>/nginx.conf` (if applicable): [https://pastebin.com/W9Ud6pFj](https://pastebin.com/W9Ud6pFj)
- Link to the exact repository being deployed (if possible/applicable):
- If a deploy is failing or behaving unexpectedly:
  - Application name - legendary-website
  - The type of application being deployed (node, php, python, ruby, etc.) - Website made in Vue and NUXT framework
  - If using buildpacks, which custom buildpacks are in use
  - If using a `Dockerfile`, the contents of that file - [https://pastebin.com/nGsn0q7v](https://pastebin.com/nGsn0q7v)
  - If it exists, the contents of your `Procfile`.
- Output of failing Dokku commands after running `dokku trace:on`
  (BEWARE: `trace:on` will print environment variables for some commands, be sure you're not exposing any sensitive information when posting issues. You may replace these values with XXXXXX):

**Comment by josegonzalez** [comment]
The `build-env` plugin was removed in 0.18.0. You can force remove the plugin by running the following commands:

```shell
rm -rf /var/lib/dokku/plugins/enabled/build-env /var/lib/dokku/plugins/available/build-env
rm -rf  /var/lib/dokku/plugins/core-enabled/build-env /var/lib/dokku/plugins/core-available/build-env
```

As for your app not responding, we're missing all the information requested regarding a deploy behaving unexpectedly. Please add all that information (its in the issue template, as well as the original post above) before we can continue debugging. Thanks.

**Comment by mikart143** [comment]
I Updated my Question. Your answer helped me, but still i Do not get my website. I am keep getting stock CentOS website. It seems that dokku do not change configuration of nginx

**Comment by josegonzalez** [comment]
Did you introduce a dockerfile after the fact? It seems as though the port mapping doesn't line up with what is in your dockerfile. You can reset this via:

```
dokku proxy:ports-set legendary-website http:80:3000
dokku nginx:build-config
```

And then your app should start working again.

**Comment by mikart143** [comment]
I added those port mappings and it still happen. I am 100% sure that app is using 3000 port

**Comment by josegonzalez** [comment]
Can you jump on our slack channel so I can provide better support via chat? Github issues is a pretty bad way to debug these sorts of issues.

**Comment by josegonzalez** [comment]
The issue here was SELinux was enabled, resulting in the nginx config files not being loaded. This is a duplicate of #3443.
